packages/components/nodes/chatmodels/AzureChatOpenAI/AzureChatOpenAI.ts

```diff
@@ -0,0 +1,134 @@
+import { OpenAIBaseInput } from 'langchain/dist/types/openai-types'
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { AzureOpenAIInput, ChatOpenAI } from 'langchain/chat_models/openai'
+
+class AzureChatOpenAI_ChatModels implements INode {
+    label: string
+    name: string
+    type: string
+    icon: string
+    category: string
+    description: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Azure ChatOpenAI'
+        this.name = 'azureChatOpenAI'
+        this.type = 'AzureChatOpenAI'
+        this.icon = 'Azure.svg'
+        this.category = 'Chat Models'
+        this.description = 'Wrapper around Azure OpenAI large language models that use the Chat endpoint'
+        this.baseClasses = [this.type, ...getBaseClasses(ChatOpenAI)]
+        this.inputs = [
+            {
+                label: 'Azure OpenAI Api Key',
+                name: 'azureOpenAIApiKey',
+                type: 'password'
+            },
+            {
+                label: 'Model Name',
+                name: 'modelName',
```

Comment:
not sure if these model names are correct. At least when checked docs - its called gpt-35-turbo. Refer to here: https://learn.microsoft.com/en-GB/azure/cognitive-services/openai/concepts/models